### PR TITLE
fix: fix undo/redo

### PR DESCRIPTION
**Pull request by TrySound**
Commands meta store introduced in latest changes polutted immerhin history. Here created separate store for client only synchronization.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: apps/builder/app/shared/commands-emitter.ts [hunk 1]
```diff
@@ -61,11 +61,14 @@ export const createCommandsEmitter = <CommandName extends string>({
       // schedule store.set to the next tick
       // so store.listen is executed after store.set below
       Promise.resolve().then(() => {
-        store.createTransaction([$commandMetas], (commandMetas) => {
-          for (const { handler, ...meta } of commands) {
-            commandMetas.set(meta.name, meta);
+        clientImmerhinStore.createTransaction(
```
Comment (kof) [inline apps/builder/app/shared/commands-emitter.ts:64]: could be just clientOnlyStore?

Comment (kof) [reply apps/builder/app/shared/commands-emitter.ts:64]: or clientSyncStore

Comment (kof) [reply apps/builder/app/shared/commands-emitter.ts:64]: Or: apiSyncstore, canvasSyncStore, pluginSyncStore

Comment (kof) [reply apps/builder/app/shared/commands-emitter.ts:64]: Wait a second, isn't it that you only want to avoid undo/redo getting synced?

Comment (kof) [reply apps/builder/app/shared/commands-emitter.ts:64]: Actually now as I think about it I don't understand why you use immerhin for shorthands at all, these things are ephemeral, real persistent things are going to happen in response to shorthands

**Review by istarkov — APPROVED** [review]

File: apps/builder/app/shared/sync/sync-stores.ts [hunk 2]
```diff
@@ -53,11 +53,13 @@ declare module "~/shared/pubsub" {
     sendStoreChanges: {
       // distinct source to avoid infinite loop
       source: SyncEventSource;
+      storeType: "client" | "server";
```
Comment (kof) [inline apps/builder/app/shared/sync/sync-stores.ts:56]: alternative names for storeType: "destination", "location", "origin"

**Comment by kof** [comment]
I tested - works now

File: apps/builder/app/shared/sync/sync-stores.ts (hunk 2 above)
Comment (TrySound) [reply apps/builder/app/shared/sync/sync-stores.ts:56]: Will get rid of it soon anyway.

**Pull request merged**